html/HookListener.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+require_once("HookNode.php");
+
+/*
+ * TODOS: This falls calls a stubbed-out method that Arthur is working on.  His
+ * work should either be added into that stubbed-out method, or remove that method
+ * and replace with his method and replace the call.
+ */
+
+if (HookNode::verifyRegisteredBroker($_SERVER['REMOTE_ADDR'])) {
+    $req = new stdClass();
+
+    foreach ($_POST as $key => $value ) {
+        $req->$key = $value;
+    }
+
+    $req->responseAddress = $_SERVER['REMOTE_ADDR'] . ':' .  $_SERVER['REMOTE_PORT'];
+
+    processRequest($req);
+} else {
+    die("PERMISSION DENIED");
+}
+
+function processRequest($request)
+{
+    if (isset($request->command)) {
+
+        switch ($request->command) {
+            case 'attachToTangle':
+                HookNode::attachTx($request);
+                break;
+            default:
+                die("UNRECOGNIZED COMMAND");
+                break;
+        }
+    } else {
+        die("NO COMMAND PROVIDED");
+    }
+}
```

Comment:
should all of this code be inside a module?

Reply:
I would be open to that.